device/components/RTA_OnDeviceComponentTask.brs

```diff
@@ -7,21 +7,7 @@ sub init()
 end sub
 
 sub setValidRequestTypes()
-	m.validRequestTypes = {
```

Comment:
Why were all of these deleted?

Reply:
It was just boilerplate that was required to add a new request. There's still a decent amount of it to add a new request but this is at least one less spot you have to touch now.